frontend/packages/dev-console/src/components/multi-tab-list/MultiTabListPage.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { match as Rmatch } from 'react-router-dom';
+import {
+  history,
+  PageHeading,
+  HorizontalNav,
+  Page,
+  ActionsMenu,
+} from '@console/internal/components/utils';
+import { ALL_NAMESPACES_KEY } from '@console/shared';
+import NamespacedPage, { NamespacedPageVariants } from '../NamespacedPage';
+import './MultiTabListPage.scss';
+
+interface MultiTabListPageProps {
+  title: React.ReactNode;
+  badge?: React.ReactNode;
+  menuActions?: any;
+  pages: Page[];
+  match: Rmatch<any>;
+}
+
+const MultiTabListPage: React.FC<MultiTabListPageProps> = ({
+  title,
+  badge,
+  pages,
+  menuActions,
+  match,
+}) => {
+  const handleNamespaceChange = (newNamespace: string): void => {
+    if (newNamespace === ALL_NAMESPACES_KEY) {
+      history.push('/pipelines/all-namespaces');
+    } else {
+      history.push('/pipelines/ns/:ns');
+    }
+  };
+  return (
+    <NamespacedPage
+      variant={NamespacedPageVariants.light}
+      hideApplications
+      onNamespaceChange={handleNamespaceChange}
+    >
+      <PageHeading
+        title={title}
+        badge={badge}
+        menuActions={menuActions}
+        className="multi-tab-list-page__heading"
+      >
+        <ActionsMenu
+          actions={menuActions}
+          title="Create"
+          actionsMenuClass="multi-tab-list-page__actions-menu"
+          toggleButtonClass="multi-tab-list-page__toggle-button"
+        />
```

Comment:
Yeah thought of using that, but I also thought why not use `ActionMenu` itself since its pretty much the same what UXD says, by just changing a bit of css. Can you please explain why using the dropdown will be better here?

Reply:
I guess a couple reasons lead me down this line of thinking.

1. Actions !== Create Button... Not sure if this is actually true or not, but the way I see actions is that they are for things like edit, and special actions like Start Pipeline - not route action to the create page
2. You had to modify `ActionMenu` to accept the styles. I brought over a Dropdown component and it seemed to accept the styles pretty easily:
```
<Dropdown
  buttonClassName="pf-m-primary"
  menuClassName="pf-m-align-right-on-md"
  title="Create"
  noSelection
  items={_.mapValues(menuActions, (v: MenuAction) => v.label || v.model.label)}
  onChange={(key: string) => {
    //...
  }}
/>
```